mangadex_downloader/upscale_coreml.py

```
@@ -0,0 +1,408 @@
+
+        try:
+            img = Image.open(input_path)
+
+            original_mode = img.mode
+            if original_mode not in ('RGB', 'RGBA'):
+                img = img.convert('RGB')
+
+            img_array = np.array(img).astype(np.float32) / 255.0
+            img_array = np.transpose(img_array, (2, 0, 1))
+
+            C, H, W = img_array.shape
```

Ruff (0.13.3)

Unpacked variable `C` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)